source/blog/2018-11-16-the-ember-times-issue-73.md

```diff
@@ -128,4 +128,4 @@ That's another wrap! ✨
 
 Be kind,
 
-Chris Ng, Jessica Jordan, Kenneth Larsen, Niels Rasmussen, Ryan Mark the crowd and the Learning Team
\ No newline at end of file
+Chris Ng, Jessica Jordan, Kenneth Larsen, Niels Rasmussen, Ryan Mark the crowd and the Learning Team
```

Comment:
```suggestion
Melanie Sumner, Chris Ng, Jessica Jordan, Kenneth Larsen, Niels Rasmussen, Ryan Mark the crowd and the Learning Team
```

Reply:
We should also add your name to L3 and add a short blurb to L12 e.g. "A teaser for Dec**Ember** 😄"...